toasty/image.py

```diff
@@ -868,12 +857,34 @@ def aspil(self):
         """
         if self._pil is not None:
             return self._pil
-        if self.mode.try_as_pil() is None:
+        if self.mode not in (ImageMode.RGB, ImageMode.RGBA):
```

Comment:
Hmm, I think that it's probably better to preserve the original semantics here. PIL *does* support non-RGB(A) image modes, [as listed here][1], and I think the most sensible behavior for this function is for it to succeed if the data array can be expressed as one of those modes.

[1]: https://pillow.readthedocs.io/en/stable/handbook/concepts.html#modes

Also, I forgot to mention this before, but as always, deleting the `try_as_pil()` function is an API break, which we should try to avoid if there's not a good reason.